### PR TITLE
v1 API

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rokamura@yahoo-corp.jp>

merge this PR after https://github.com/vdaas/vald/pull/740 is merged

### CODE REVIEW

**Review by kmrmt — APPROVED** [review]
LGTM

**Comment by datelier** [comment]
@rinx Do you think you need to write "Requirements" in README.md?

**Comment by rinx** [comment]
definitely we need it.
I think it is better to write "Install", "Usage" and "Example" sections in README.md. (also for other client libs)
okay, i'll prepare them before it becomes ready to merge.

**Review by datelier — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,3 +2,61 @@
...
 [![PyPI version](https://badge.fury.io/py/vald-client-python.svg)](https://badge.fury.io/py/vald-client-python)
 
+A Python gRPC client library for [Vald](https://github.com/vdaas/vald).
+
+## Install
+
+Using pip,
+
+```sh
+pip install grpcio-tools
```
Comment (kmrmt) [inline README.md]: If the users just want to use vald-client-python, they need not to install grpcio-tools.

**Review by kmrmt — CHANGES_REQUESTED** [review]

**Comment by kmrmt** [comment]
I forgot to need to fix setup.cfg#L22
We need to use `find_namespace` directive instead of `find`

File: README.md (hunk 1 above)
Comment (rinx) [reply README.md]: Thanks

**Comment by rinx** [comment]
@kmrmt Could you please send me a code suggestion about it?

**Review by kmrmt — APPROVED** [review]

**Pull request merged**